guides/app-hooks.md

```diff
@@ -4,30 +4,67 @@
 Project Firefly applications created with [our CLI](https://github.com/adobe/aio-cli) are inherently npm packages.
 This means they support many of the npm conveniences node developers expect.
 
-package.json may include the following scripts which are triggered at various times while an app is being built, deployed and/or run.
+`package.json` may include the following scripts which are triggered at various times while an app is being built, deployed and/or run.
 
 ## Sample hooks configuration
 
-For example, pre and post hooks can be defined for the `run`, `build` and `deploy` operations in the package.json file of your app as follows:
+For example, pre and post hooks can be defined for the `run`, `build` and `deploy` command in the package.json file of your app. For the `build` and `deploy` commands, you can specify build and deploy script substitutes as well.
```

Comment:
Typo: `commands in the package.json file` instead of `command in the package.json file`